pkgs/applications/audio/fverb/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, fetchFromGitHub
+, faust2lv2
+}:
+stdenv.mkDerivation rec {
+
+  pname = "fverb";
+  # no release yet:
+  version = "unstable-2020-04-04";
```

Comment:
There _is_ a release now: https://github.com/jpcima/fverb/releases/tag/bells but that tag is horrible for automatic updates etc so I opened an issue upstream: https://github.com/jpcima/fverb/issues/2

Reply:
Thanks!

Reply:
As discussed in that issue, it was not a release.
I have updated to the latest commit now.

Slightly related, because te new version has a makefile: https://github.com/jpcima/fverb/issues/4#issuecomment-641498128:
Do you know why some packages need
``  makeFlags = [ "PREFIX=$out" ];`` and some need ``  makeFlags = [ "PREFIX=${placeholder "out"}" ];``?
Most packages don't need any of that, because nix sets PREFIX automatically, iiuc.

Reply:
> Do you know why some packages need ...

It's likely that all of the options will work for these packages. The difference is a subtlety:

- `PREFIX=$out` is interpreted by the shell, thanks to the fact that `$out` is an environmental variable available during the build.
- `${placeholder "out"}` is evaluated by Nix it self, during the evaluation of all strings in the arguments of `mkDerivation`.
- `PREFIX=$(out)` (you didn't list this option) is evaluated by `make`, and it works thanks to the fact that `$out` is an environmental variable available during build.

> Most packages don't need any of that, because nix sets PREFIX automatically

I don't know why is that, but it's a bit common to do that.

Reply:
Thanks for explaining.

> It's likely that all of the options will work for these packages. 

No, that is not the case, for example this package did not work with the first option you listed.

> I don't know why is that, but it's a bit common to do that.

Not sure if I understand. You mean you don't know why some packages do and some don't need PREFIX to be set manually?
Yeah, that *is* common, and I don't get it.

Reply:
You mean you don't know why some packages do and some don't need PREFIX to be set manually?

Yes.

I _think_ that stdenv's builder sets the prefix only via `./configure --prefix` and not straight when calling `make` - maybe that's the reason only some packages need `PREFIX` to be set manually.

Reply:
OK, thanks.